packages/amplify_authenticator/lib/src/widgets/form_fields/sign_up_form_field.dart

```diff
@@ -237,10 +262,84 @@ abstract class SignUpFormField<FieldValue> extends AuthenticatorFormField<
         field: SignUpField.custom,
         validator: validator,
         attributeKey: attributeKey,
+        required: required,
       );
 
   /// Custom Cognito attribute key.
   final CognitoUserAttributeKey? _customAttributeKey;
+
+  @override
+  int get priority {
+    switch (field) {
```

Comment:
Is priority descriptive enough? Maybe something like `displayPriority`?  If you feel strongly I'm fine with this as is.

Reply:
I agree - I prefer `displayPriority`. Updated.